### ISSUE TITLE
[BUG] Broken links

### ISSUE THREAD

**Issue by jribbink** [onflow/docs#328]
### Current Behavior

https://developers.flow.com/tools/bridges

### Expected Behavior

https://developers.flow.com/tools/bridges

### Steps To Reproduce

https://developers.flow.com/tools/bridges

### Environment

```markdown
- OS:https://developers.flow.com/tools/bridges
- Node:
- npm:
```

**Comment by gregsantos** [comment]
Thanks @jribbink hit me with an update on these when you can.
Have you started to tackle adding to vercel redirects or you think other issue at play?

**Comment by gregsantos** [comment]
Think some of these can be redirected now to the new node stuff under Reference.
Some are done and those remaining, sounds like should redir to home.

Can redirect bridges one once `https://developers.flow.com/overview` stuff is moved

**Comment by jribbink** [comment]
It looks like https://developers.flow.com/guides/nft-marketplace#introduction is the new home of the in-dapp-payments stuff.  I'm not a big fan of how this page is laid out right now honestly.  It's hard to follow and I think that in-dapp-payments deserved its own page & could have been called out in the NFT Marketplace guide.

I don't think in-dapp payment falls under the umbrella of NFT marketplace & FIAT onramps are a broader concept than just NFT marketplaces alone.

**Comment by Aliserag** [comment]
I agree with you, I asked Jacob if he can copy and redo the NFT-marketplace guides into Emerald Academy so we can remove it from our site and he said he will start doing it their next cycle. I think this is more relevant to DeFi and we should have our own section relating to it under overview.
